net/uradvd/Makefile

```diff
@@ -0,0 +1,48 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=uradvd
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL=https://github.com/freifunk-gluon/uradvd.git
+PKG_SOURCE_DATE:=2021-08-14
+PKG_SOURCE_VERSION:=9b0da60e27c67305d251b10163e388191d566d7a
+PKG_MIRROR_HASH:=70dac4c156cd98baea92c9f1fb14692737b6c55c2aa0442c272ec9abe8fe06e4
+
+PKG_MAINTAINER:=Moritz Warning <moritzwarning@web.de>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=LICENSE
+
+PKG_BUILD_DIR := $(BUILD_DIR)/$(PKG_NAME)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/uradvd
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=Advertise an IPv6 prefix/route
+endef
+
+define Package/uradvd/description
+	Advertise an IPv6 prefix from the node
+endef
+
+define Build/Prepare
+	mkdir -p $(PKG_BUILD_DIR)
+	$(CP) ./src/* $(PKG_BUILD_DIR)/
+endef
```

Comment:
You can drop the whole Prepare step.

Reply:
Done. But the package does not compile right now:
```
make[4]: Entering directory '/home/mwarning/openwrt/build_dir/target-mipsel_24kc_musl/uradvd'
make[4]: *** No targets specified and no makefile found.  Stop.
make[4]: Leaving directory '/home/mwarning/openwrt/build_dir/target-mipsel_24kc_musl/uradvd'
make[3]: *** [Makefile:41: /home/mwarning/openwrt/build_dir/target-mipsel_24kc_musl/uradvd/.built] Error 2
make[3]: Leaving directory '/home/mwarning/openwrt/package/uradvd'
time: package/uradvd/compile#0.86#0.19#1.58
    ERROR: package/uradvd failed to build.
```

Any idea?

Reply:
Remove the `PKG_BUILD_DIR` line from the Makefile to fix this error.